integration_tests/assets/docker-compose.yml

```diff
@@ -57,4 +57,4 @@ services:
       # - "${LOCAL_GIT_REPOS}/wazo-provd-client/wazo_provd_client:/usr/local/lib/python3.7/site-packages/wazo_provd_client"
       # - "${LOCAL_GIT_REPOS}/xivo-bus/xivo_bus:/usr/local/lib/python3.7/site-packages/xivo_bus"
       # - "${LOCAL_GIT_REPOS}/xivo-dao/xivo_dao:/usr/local/lib/python3.7/site-packages/xivo_dao"
-      # - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/usr/local/lib/python3.7/site-packages/xivo"
+      - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/usr/local/lib/python3.7/site-packages/xivo"
```

Comment:
Did you mean to leave this not commented?

Reply:
Yeah ... so annoying to uncomment each time :stuck_out_tongue_closed_eyes: